semperfiwebdesign/all-in-one-seo-pack
# Make WC upgrade button orange

Issue #2405

## Proposed changes

Makes the Upgrade button for the WooCommerce notice orange. Also affects Upgrade button on the Welcome screen as I've improved the styling a bit so it looks less like a flat pancake.

## Types of changes

What types of changes does your code introduce?

- Improves existing functionalty

## Checklist


## Testing instructions

Check the Upgrade button both on the Welcome screen and in the WooCommerce notice on a new* website and make sure it looks good. 

_*on an existing website the notice might have been generated already which is why the Upgrade button will be missing the .button-orange class._

## Further comments

None.
